packages/dds/merge-tree/src/revertibles.ts

```diff
@@ -281,11 +281,28 @@ function revertLocalInsert(
 	}
 }
 
+/**
+ * Revertibles are new and require the option
+ * mergeTreeUseNewLengthCalculations to be set as true on the underlying merge tree
+ * in order to function correctly.
+ * @alpha
+ */
+export function revertMergeTreeDeltaRemoveRevertible(
+	driver: MergeTreeRevertibleDriver,
+	revertible: MergeTreeDeltaRevertible,
+) {
+	assert(revertible.operation === MergeTreeDeltaType.REMOVE, "revertible must be remove");
+	const mergeTreeWithRevert = findMergeTreeWithRevert(revertible.trackingGroup.tracked[0]);
+	return revertLocalRemove(driver, mergeTreeWithRevert, revertible, true);
+}
+
 function revertLocalRemove(
 	driver: MergeTreeRevertibleDriver,
 	mergeTreeWithRevert: MergeTreeWithRevert,
 	revertible: TypedRevertible<typeof MergeTreeDeltaType.REMOVE>,
+	returnNewSegments: boolean = false,
 ) {
+	const newSegments: ISegment[] | undefined = returnNewSegments ? [] : undefined;
```

Comment:
I'd rather not see revertMergeTreeDeltaRemoveRevertible exported. can't you capture these by wrapping the call in an event handler? something like

```
const newSegs=[];
const captureSegs=(args)=>args.deltaSegments.foreach((d)=>newSegs.push(d.segment));
sequence.on("sequenceDelta",captureSegs)
revertMergeTreeDeltaRevertibles(...)
sequence.off("sequenceDelta",captureSegs)
```